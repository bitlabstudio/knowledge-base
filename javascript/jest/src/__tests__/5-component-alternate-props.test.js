import React from 'react'
import renderer from 'react-test-renderer'

import TestComponent from '../testedFiles/5-component-alternate-props'

describe('TestCompont', () => {
  it('renders the correct title', () => {
    const component = renderer.create(<TestComponent title='Foobar' />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders a different title', () => {
    const component = renderer.create(<TestComponent title='Different Title' />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
