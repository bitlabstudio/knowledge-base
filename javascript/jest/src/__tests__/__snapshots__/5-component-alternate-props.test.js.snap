// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestCompont renders a different title 1`] = `
<div>
  <h1>
    Different Title
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Animi assumenda consequuntur distinctio enim eos excepturi expedita fugiat impedit iure natus necessitatibus nemo nulla perferendis quia quis saepe, totam ut vel!
  </p>
</div>
`;

exports[`TestCompont renders the correct title 1`] = `
<div>
  <h1>
    Foobar
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Animi assumenda consequuntur distinctio enim eos excepturi expedita fugiat impedit iure natus necessitatibus nemo nulla perferendis quia quis saepe, totam ut vel!
  </p>
</div>
`;
